flask_app.py

```diff
@@ -19,9 +28,33 @@
 from models import Employee
 
 app = Flask(__name__)
+load_dotenv()
 app.secret_key = os.getenv("SECRET_KEY", "dev")
+
+login_manager = LoginManager()
+login_manager.init_app(app)
+login_manager.login_view = "login"
+login_manager.login_message_category = "info"
+
+ADMIN_USERNAME = os.getenv("ADMIN_USERNAME", "admin")
+ADMIN_PASSWORD = os.getenv("ADMIN_PASSWORD", "test123")
+
+
+class SimpleUser(UserMixin):
+    def __init__(self, user_id: int, username: str):
+        self.id = user_id
+        self.username = username
+
+
+SINGLE_USER = SimpleUser(1, ADMIN_USERNAME)
+
+
+@login_manager.user_loader
+def load_user(user_id: str):
+    return SINGLE_USER if str(SINGLE_USER.id) == str(user_id) else None
```

Comment:
`return SINGLE_USER if user_id == str(SINGLE_USER.id) else None` -> user_id is already a string so no need to use str() here